internal/subscriber/subscriberimpl_test.go

```diff
@@ -306,7 +306,7 @@ func getMockConsumer(ctx context.Context, ctrl *gomock.Controller) *mockMB.MockC
 		Topic:     topicName,
 		Partition: partition,
 	}
-	cs.EXPECT().GetTopicMetadata(ctx, req).Return(messagebroker.GetTopicMetadataResponse{}, nil).AnyTimes()
+	cs.EXPECT().GetTopicMetadata(gomock.Any(), req).Return(messagebroker.GetTopicMetadataResponse{}, nil).AnyTimes()
```

Comment:
why have we changed this..?

Reply:
I am also using getMockConsumer func in my test case, and there GetTopicMetadata expect context with Cancel that why added gomock.Any(). if i create the same func with the small change there will code duplicacy.

Reply:
what about this..?